### ISSUE TITLE
Feature Request: Ability to change ingredient (and output) amount on processing patterns in wireless pattern encoding terminal

### ISSUE THREAD

**Issue by Meryl** [Mari023/AE2WirelessTerminalLibrary#300]
The vanilla pattern encoding terminal allows you to change amounts in a processing recipe by middle clicking the slot (either one of the input slots or an output slot). This feature would be really nice to have on the wireless version as well.

**Comment by Mari023** [comment]
the wireless pattern encoding terminal uses the same code as the pattern encoding terminal, this *should* work

**Comment by Meryl** [comment]
**TLDR:** This doesn't work in the (Universal) Wireless Terminal due to a conflict with a bound middle click for Inventory Tweaks Renewed. It does work for the regular pattern encoding terminal, so it's weird that it would conflict for one but not the other.

The part of the post I mostly typed up before I found the cause:
------------

Here are the versions I'm running (part of ATM9TTS):

|Mod|version|
|----|----|
|Applied Energistics 2|15.2.11|
|AE2 Wireless Terminal Library|1.3.0|
|Inventory Tweaks Renewed|1.1.0|

There's also a bunch of AE add-ons etc., here's the ones I could find:

|Mod|version|
|----|----|
|AE2 Things|1.2.1|
|AE2 Insert Export Card|1.3.0|
|AECapFix|0.9|
|Applied Botanics|1.5.0|
|AppliedE|0.14.0|
|AppliedFlux|1.1.8|
|Applied Mekanistics|1.4.2|
|ExtendedAE|1.1.11|

It does indeed hint that it should be possible: 
![image](https://github.com/user-attachments/assets/269c05a8-74df-4647-9ae8-1bb0be4ba67f) (the amount of 4 in this screenshot is because I had 4 of the item on my cursor when I clicked the ingredient slot)

I just checked key binding conflicts, and it turns out that Inventory Tweaks Renewed has the option "Sort Inventory Under Cursor" which is conflicting for the (Universal?) Wireless Terminal only.

**Comment by Mari023** [comment]
ah, either ae2 or Inventory Tweaks Renewed has something to fix the conflict for the pattern access terminal, but not for the wireless one
I'll look into it

**Comment by Meryl** [comment]
That's great, thanks!
